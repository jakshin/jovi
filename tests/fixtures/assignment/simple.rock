(Common variables)
The thing is a very bad thing
The darkness is the right place for it
Put the thing into the darkness
Say the darkness (1435)

(Proper variables)
That was confusing
This is a simple explanation
Put This into That
Say That (161)

(Literals)
Put null into Something Say Something
Put true into Something Say Something
Put 42 into Something Say Something
Put 12.34 into Something Say Something
Put "foo" into Something Say Something
Put mysterious into Something Say Something
