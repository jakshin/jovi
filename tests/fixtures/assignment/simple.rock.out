1435
161
null
true
42
12.34
foo
mysterious
